sphinx_panels/__init__.py

```diff
@@ -60,7 +60,13 @@ def update_css(app: Sphinx):
 
     # Add core CSS
     css_files = [r for r in resources.contents(css_module) if r.endswith(".css")]
-    if not app.config.panels_add_boostrap_css:
+    if app.config.panels_add_boostrap_css:
+        LOGGER.warning(
+            "`add_boostrap_css` will be deprecated. Please use `add_bootstrap_css`."
+        )
+        app.config.add_bootstrap_css = app.config.panels.add_boostrap_css
```

Comment:
want happens here though if someone sets `panels_add_bootstrap_css=False`?
won't this override that?

Reply:
ah that's a good point, I forgot it defaults to `True`. I'm not sure how to check for whether a configuration value has been provided by a user or not :-/

Reply:
perhaps `add_boostrap_css` and `panels_add_bootstrap_css` should be set to `None` initially, to check if someone actually sets a value on them.

Then: 

1. warn if `add_boostrap_css` is not None
2. fail if both are not None
3. If one is not None, take that value
4. default to `True`

Reply:
well I guess if they both default to `True`, then can't we assume that if *either* of them is `False`, then they should both be `False`? So the check would become:

```python
    if not app.config.panels_add_boostrap_css:
        LOGGER.warning(
            "`panels_add_boostrap_css` will be deprecated. Please use `panels_add_bootstrap_css`."
        )
        app.config.panels_add_bootstrap_css = app.config.panels_add_boostrap_css
```

Reply:
Well my way is more rigorous, you decide 🤷